dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -36,14 +35,25 @@ RUN ln -s -f /usr/bin/gcc-8 /usr/bin/arm-linux-gnueabihf-gcc
 RUN ln -s -f /usr/bin/gcc-8 /usr/bin/aarch64-linux-gnu-gcc
 {% endif %}
 
+# TODO: Remove these lines once restore_neighbors.py is converted to Python 3
+RUN apt-get install -f -y python-dev python-pip
+RUN pip2 install --upgrade 'pip<21'
+RUN apt-get purge -y python-pip
+RUN pip2 install setuptools==40.8.0
+RUN pip2 install wheel==0.35.1
```

Comment:
why do we still need pip2 in docker-orchagent?

Reply:
We need Python 2 in docker-orchagent until restore_neighbors.py is converted to Python 3, currently being worked on by @zhenggen-xu.

Reply:
Removed, as Zhenggen took care of this in https://github.com/Azure/sonic-buildimage/pull/6207